rts/Game/Camera/SpringController.cpp

```diff
@@ -66,7 +87,7 @@ void CSpringController::MouseMove(float3 move)
 	const bool moveFast = camHandler->GetActiveCamera()->GetMovState()[CCamera::MOVE_STATE_FST];
 
 	move *= 0.005f;
-	move *= (1 + moveFast * 3);
+	move *= (1 + moveFast * 3); // See on siin juba muutunud, aga *3 on ikkagi HARD-CODED
```

Comment:
Would you want to apply the configurable multiplier here?

Reply:
Sorry, messed up something here. Since the comment is Estonian, it was not meant to be pushed. (Ofc, will change to varibale here)